server/GlassAuth.cs

```diff
@@ -227,15 +227,18 @@
           echo("Glass Error: " @ %this.buffer);
       }
 
-			%object.schedule(0,delete);
-
+		%object.schedule(0,delete);
 		} else {
 			echo("Glass Server Auth: \c2INVALID RESPONSE");
       echo("Glass Error: " @ %this.buffer);
 
-      GlassAuthS.authing = false;
-      GlassAuthS.schedule(10*1000, reident);
-		}
+      } else {
+          echo("Glass Server Auth: \c2INVALID RESPONSE");
+          echo(%this.buffer);
+
+          GlassAuthS.authing = false;
+          GlassAuthS.schedule(10*1000, reident);
```

Comment:
... and so is it here.

Reply:
Corrected to the best of my knowledge -- but there appear to be some issues unrelated to the changes in this PR.

I noticed some other confusing indentation within GlassAuth.cs. The entire file seems to be using tabs and spaces interchangeably (ex: try highlighting [lines 4 and 5](https://github.com/BlocklandGlass/BlocklandGlass/blob/development/server/GlassAuth.cs#L4)). May be worth fixing in a separate branch?

In the meantime, I've reverted the indentation to otherwise appear as I found it, rather than trying to bulk-fix it here. https://github.com/BlocklandGlass/BlocklandGlass/pull/276/files#diff-7f789ccb4c4196de405ab8504fc69449a8bdc3c46ae2c6f7cf45212d7235ec91